.rubocop.yml

```diff
@@ -7,7 +7,7 @@ AllCops:
 
 # This should go down over time.
 Metrics/ClassLength:
-  Max: 330
+  Max: 342
```

Comment:
It would be better to disable the cop on the specific class rather than increasing this limit, as we want this number to go down and not up, and the cop comment helps to highlight the classes which break this limit for later refactoring.